### ISSUE TITLE
Improve Documentation

### ISSUE THREAD

**Issue by deepanshu-rawat6** [upes-open/OSoC-bio.lnk#17]

**Comment by karthiknadar1204** [comment]
I would like to work on this issue under osoc'23.
Here i propose to make the readme elaborate by adding an elaborate way to open a pr and set up the project as well as shell commands so that anyone starting out with the project can start out easily

**Comment by deepanshu-rawat6** [comment]
Yes @karthiknadar1204 , I'll be happy to assign you this issue. But, we'll stress on creating documentation in the last three days of the project, so we can't really assign this issue as of now.

Sure, you can try to contribute via others issue, best of luck✨

**Comment by karthiknadar1204** [comment]
Ah,i insist on adding the process to open a pr as any new contributor might have a hard time navigating ,should I open a separate issue then?

**Comment by deepanshu-rawat6** [comment]
Yes @karthiknadar1204 , I'll create an issue and then assign it to you. Make sure you create a proper descriptive ```CONTRIBUTING.MD``` file in the ```root``` of this repository, along with some **images** which would help new contributors. 

Plus point: Add some git commands, some more insights on fetching and pull changes
